webapp/tests/forms/flows/test_lotse_step_chooser.py

```diff
@@ -0,0 +1,22 @@
+import pytest
+
+from app.forms.flows.lotse_step_chooser import LotseStepChooser
+from app.forms.steps.lotse.confirmation_steps import StepSummary
+from app.forms.steps.lotse.lotse_steuerlotse_steps import StepSteuernummer
+
+
+class TestLotseStepChooserInit:
+    @pytest.fixture(autouse=True)
+    def attach_fixtures(self, test_request_context):
+        self.req = test_request_context
+
+        self.testing_steps = [
+            StepSteuernummer,
+        ]
+        self.endpoint_correct = "lotse"
+
+    def test_set_attributes_correctly(self):
```

Comment:
I would question whether we need this test at all. It mainly tests functionality of the base class. Just throwing it out there for you to consider.

Reply:
I agree